### PR TITLE
Change .NET implementations overview document

### LINKED ISSUE #105: Suggestions for Overview of .NET Implementations topic
* The opening paragraph seems very vague. If I'm a newbie or non-developer and somehow land on this topic, what do I use the .NET platform and these standards for?

.NET Core section
* You introduce the list of characteristics like this "There are several characteristics of .NET Core:", but you're actually listing the main ones. Maybe it'd be good to be more specific.
* I'd use a more common word than boon. Maybe advantage or gain?
* The list of characteristics doesn't look uniform. Some are written as paragraphs, some as list items. It'd be good to have a consistent behavior.
* For cross-platform support, is it really about running on the best environment? Or is it about the different environment offerings we have out there?
* For open source: Should we also mention the transparency aspect of open source? Or having the community helping shape the future of .NET Core? The description seems really minimal.
* For application isolation, instead of "take" wouldn't be better to say embed?

.NET Framework
* It should be .NET Framework Class Library not Framework Class Libraries. Also, should it link to the library on MSDN?
* I would avoid using the apostrophe (') there to show possession. So instead of "Windows Forms' strength" I'd say "The strength of Windows Forms" and for the second one, I'd just remove the apostrophe.
* "that is suited for your use case" or "that is best suited for your use case"?
* I don't get what MVC does based on the description. Should we talk about patterns or separation of concerns here? The link you point to has a good brief description

.NET native
* fix punctuation here: (such as C#, Visual Basic, F# etc.)
* put (IL) after "Intermediate Language"
* Maybe this sentence: "IL is then picked up by the runtime, and Just-In-Time compiled at run-time to machine code." could be better explained as "At run-time, a just-in-time (JIT) compiler translates the IL into native code" (or machine code)
* .NET Native NET Native - remove duplication
* Should we link to https://msdn.microsoft.com/en-us/library/windows/apps/dn726767.aspx, where you mention UWP apps?

### PR DESCRIPTION
**Pull request by thesmithee**
Change the things suggested in #105. Also add the document to the new TOC.
Move and rename the document from getting-started/overview.md to
concepts/editions-overview.md for more clarity.

Fix #105 

/cc @mairaw

### CODE REVIEW

File: docs/concepts/editions-overview.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
...
 
-There are several characteristics of .NET Core:
+The following are main characteristics of .NET Core:
 
-**Cross-platform support** is the first important feature. For applications, it is important to use those platforms that will provide the best environment for their execution. Thus, having an application platform that can enable the app to be ran on different operating systems with minimal or no changes provides a significant boon.
+**Cross-platform support** is the first important feature. For applications, it is important to use those platforms that will provide the best environment for their execution. Thus, having an application platform that can enable the app to be ran on different operating systems with minimal or no changes provides a significant gain.
 
-**Open Source** because it is proven to be a great way to enable a larger set of platforms, supported by community contribution.
+**Open Source** because it is proven to be a great way to enable a larger set of platforms, supported by community contribution. It is also a very open way of developing software which enables better community inclusion in the overall process of designing .NET Platform.
```
Comment (matthewreily) [inline docs/concepts/editions-overview.md]: change "designing .NET platform" to "designing **the** .NET Platform"

File: docs/concepts/primer.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ By [Rich Lander](https://github.com/richlander), [Zlatko Knezevic](https://githu
 
 Multiple implementations of .NET are available, based on open [.NET Standards](https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/dotnet-standards.md) that specify the fundamentals of the platform. They are separately optimized for different app types (e.g. desktop, mobile, gaming, cloud) and support many chips (e.g. x86/x64, ARM) and operating systems (e.g. Windows, Linux, iOS, Android, OS X). Open source is also an important part of the .NET ecosystem, with multiple .NET implementations and many libraries available under OSI-approved licenses.
 
-You can take a look at the [Overview of .NET implementations](../getting-started/overview.md) document to figure out all of the different editions of .NET Framework that are available, both Microsoft’s and others.
+You can take a look at the [Overview of .NET implementations](editions-overview.md) document to figure out all of the different editions of .NET Framework that are available, both Microsoft’s and others.
```
Comment (matthewreily) [inline docs/concepts/primer.md]: same as above add a **the** prior to ".NET Framework".

File: docs/concepts/editions-overview.md [hunk 3]
```diff
@@ -6,11 +6,11 @@
 
 .NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently supports three main operating systems: Linux, Windows and OS X.
 
-There are several characteristics of .NET Core:
+The following are main characteristics of .NET Core:
```
Comment (matthewreily) [inline docs/concepts/editions-overview.md]: I think this should be "The following are **the** main characteristics of **the** .NET Core."

**Comment by matthewreily** [comment]
LGTM with just a couple of small nitpick changes.

File: docs/concepts/editions-overview.md [hunk 4]
```diff
@@ -30,14 +30,14 @@ There are additional stacks built on top the .NET Framework that allow developer
 
 ## .NET Native
 
-.NET Native is not so much an “edition” of the .NET platform as it is a set of tools that allow developers to have different build outputs. The normal .NET source compilation process takes the source code written in one of the .NET languages (such as C#, Visual Basic, F# etc.) and produces something called “Intermediate Language”. IL is then picked up by the runtime, and Just-In-Time compiled at run-time to machine code.
+.NET Native is not so much an"edition" of the .NET platform as it is a set of tools that allow developers to have different build outputs. The normal .NET source compilation process takes the source code written in one of the .NET languages (such as C#, Visual Basic, F# etc.) and produces something called "Intermediate Language" (IL). At run-time, a just-in-time (JIT) compiler compiles the IL into machine code.
```
Comment (mairaw) [inline docs/concepts/editions-overview.md]: Missing space before "edition"

File: docs/concepts/editions-overview.md (hunk 3 above)
Comment (mairaw) [inline docs/concepts/editions-overview.md]: When I see this list, it almost makes me thing it's a list of advantages over .NET Framework. Is there anything else that we should list if I'm considering switching to .NET Core coming from other platforms? Choice of languages? ASP.NET v5/MVC? Etc.

File: docs/concepts/editions-overview.md (hunk 4 above)
Comment (mairaw) [inline docs/concepts/editions-overview.md]: Missing comma after F# in "(such as C#, Visual Basic, F#, etc.)"

**Comment by mairaw** [comment]
Went in and reviewed it again against my comments from issue #105. Left a couple of feedback notes on the topic itself.

Here's what I think is still pending:
- The opening paragraph seems very vague. If I'm a newbie or non-developer and somehow land on this topic, what do I use the .NET platform and these standards for?
- The list of characteristics doesn't look uniform. Some are written as paragraphs, some as list items. It'd be good to have a consistent behavior.

Let me know if you disagree.

Thanks @blackdwarf!

**Comment by richlander** [comment]
Would be good to review this once the other comments are addressed.

File: docs/concepts/editions-overview.md (hunk 3 above)
Comment (blackdwarf) [inline docs/concepts/editions-overview.md]: There are some, but I don't want to actually add them here. Trying for the "impassionate overview" style & mood. :)

**Comment by blackdwarf** [comment]
@mairaw for the 2nd point above, not sure I'm seeing it. Can you point me to an example? Or do you mean characteristics between the editions?

**Comment by mairaw** [comment]
@blackdwarf 
Let me see if I can make that clearer. For example, this would be one way to make it look consistent:

**Cross-platform support** - for applications, it ....
**Open Source** - it is proven to be a great way to enable a larger set of platforms, supported ...
**Better packaging story** - the framework is distributed as a set of packages ...

Because you have some with plain paragraphs and some looking like list items. Makes sense?

File: docs/concepts/editions-overview.md (hunk 3 above)
Comment (mairaw) [inline docs/concepts/editions-overview.md]: Meaning you don't wanna sell too much, just a bit? :)

Comment (blackdwarf) [inline docs/concepts/editions-overview.md]: A leetle bit. :)

Z

**Comment by blackdwarf** [comment]
@mairaw I...guess? :))

**Comment by blackdwarf** [comment]
Sorry it took so long, @mairaw comments are addressed.

**Comment by richlander** [comment]
See #208. Instead of provide more feedback, I made changes directly. Since this PR had merge conflicts, I also rebased the changes.

/cc @blackdwarf

**Comment by richlander** [comment]
Closing due to #208 landing.

**Pull request closed without merging**